archunit-junit/junit5/engine/src/main/java/com/tngtech/archunit/junit/internal/ArchUnitTestDescriptor.java

```diff
@@ -56,7 +56,7 @@ class ArchUnitTestDescriptor extends AbstractArchUnitTestDescriptor implements C
     private ClassCache classCache;
 
     private ArchUnitTestDescriptor(ElementResolver resolver, Class<?> testClass, ClassCache classCache) {
-        super(resolver.getUniqueId(), testClass.getSimpleName(), ClassSource.from(testClass), testClass);
+        super(resolver.getUniqueId(), testClass.getName(), ClassSource.from(testClass), testClass);
```

Comment:
Changed this because Jupiter would group field-flavored and method-flavored test results under different parent nodes (one under FQ class name and the other under simple class name)